packages/yoroi-extension/app/connector/components/signin/cardano/SignTxSummaryComponent.js

```diff
@@ -0,0 +1,45 @@
+import { Box, Typography } from '@mui/material';
```

Comment:
1. Missing the `// @flow` flag 
  ```suggestion
  // @flow 
  import { Box, Typography } from '@mui/material';
  ```
2. Use the `SignTxSummary` as the component name instead of `SignTxSummaryComponent`